src/main/java/frc/robot/RobotContainer.java

```diff
@@ -47,7 +47,7 @@ public RobotContainer() {
    */
   private void configureBindings() {
     // Triggers are a thing that we might need to use so keep that in mind
-    driver.b().onTrue(new ArmCommand(armSubsystem));
+    //driver.b().onTrue(new ArmCommand(armSubsystem));
```

Comment:
Resolve bug with #42 instead of just commenting out the code that causes the error.